streamalert_cli/test/handler.py

```diff
@@ -234,7 +234,7 @@ def __init__(self, options, config):
             'AWS_ACCOUNT_ID': self._config['global']['account']['aws_account_id'],
             'ALERTS_TABLE': '{}_streamalert_alerts'.format(prefix),
         }
-        if options.stats:
+        if 'stats' in options:
```

Comment:
why this change? `options` is an argparse Namespace object.. not a dict

Reply:
Otherwise I get an error `AttributeError: 'Namespace' object has no attribute 'stats'`
while running  `python manage.py test live` or  `python manage.py test classifier`